include/mpd/idle.h

```diff
@@ -89,6 +89,9 @@ enum mpd_idle {
 
 	/** a message on a subscribed channel was received */
 	MPD_IDLE_MESSAGE = 0x400,
+
+	/** a partition was added or changed */
```

Comment:
The MPD documentation says you could also perceive a removal event on a partition. I am not sure how you would achieve that, as there is no command to remove a partition.

"partition: a partition was added, removed or changed"
reference link: https://www.musicpd.org/doc/html/protocol.html

Reply:
The partition system is not yet finished in MPD. One day, we need a command to remove a partition.
I have a patch in my stgit branch which adds the `moveoutput` command to the protocol, to move an existing output to another partition; without this, partition don't make sense. But finishing this implementation is hard, and other things keep distracting me.
Anyway, thanks for your work, I hope this serves as a motivation for me to finish the feature!

Reply:
Ooh i see. thanks for the clarification